css/styles.css

```diff
@@ -35,7 +35,7 @@
 
 
 body {
-  font-family: Montserrat;
+  font-family: Montserrat, sans-serif, arial; 
```

Comment:
maybe change the order of sans-serif and arial.

Reply:
Nice! Great feedback @felixfkhk